### ISSUE TITLE
///,TAB does nothing

### ISSUE THREAD

**Issue by jamezpolley** [giampierod/gpd.atom#5]
Instructions say:

    1. Create new file, give it the extension .GPD, and open it in Atom.
    2. Type ///,Tab. It will instantly give you the section layouts and put your cursor in the Backlog section.

I've done that, but nothing is happening. Is there some other way I can trigger whatever magic is meant to happen here?

Similarly, typing # then pressing tab does nothing, and so on. I suspect there's some kind of snippet or autocomplete library I need?

**Comment by giampierod** [comment]
Does the language detector at the bottom right say "GPD" or something else? I haven't changed the snippets in a long time, so I am not sure what is happening. 

What version of Atom are you using?

**Comment by giampierod** [comment]
Quick update. The last fix I made to Snippets was at the beginning of December. It looks like those fixes were included in the 0.13.0 release. I have pushed out a 0.14.1 release which has my latest working version that I have been using on my machine for the last few months where I know the snippets work. Please update and let me know if the problem persists.

**Comment by giampierod** [comment]
@jamezpolley Did you give it a try? Is it working for you now?

**Comment by jamezpolley** [comment]
Sorry, didn't see the notifications until now :(

Language detector says GPD. Atom version is 1.8.0, and I've just installed 0.14.1 of GPD. Still nothing.

Do I need some other package to make the snippets work?

.... oh wait. I just looked at the settings and realised that the defined snippet is not '///,' it's '///'. So the instruction to type '///,Tab' should be read as typing a literal '///' followed by a tab - no , needed. Once I tried it without the , this worked as expected.

Apparently the problem here is that the user can't understand the instructions. I'll offer myself a round of applause and go in search of more coffee.
